### DISCUSSION TITLE
Details not shown in Activity Triggers

### DISCUSSION THREAD

**Discussion by paulprivalgo** [microsoft/DurableFunctionsMonitor#135] [Q&A] [answered]
Hello

When I look at the executions history part of a single orchestration function I can see the following columns:

Timestamp
EventType
EventId
Name
ScheduledTime
Result 
Details

How can I get something to show in Details?

Is details designed to show the input of the activity function?

Paul

**Comment by scale-tone** [comment, marked as answer]
Hi @paulprivalgo , that field is intended for failure details, when e.g. an activity or a suborchestration fails. Like here, for example:

![image](https://github.com/microsoft/DurableFunctionsMonitor/assets/5447190/937e2b08-131b-4adf-9e81-c2b488ea26f6)
